### DISCUSSION TITLE
No files matching the pattern 'users' error

### DISCUSSION THREAD

**Discussion by psalmuel19** [cjbarrie/academictwitteR#207] [Q&A] [answered]
Hi all,
I am trying to bind the json files I collected using the academictwitteR package. I had some successes with binding but later it began returning an error. The directory is correct as it was the file I used earlier but without binding to an object. The code and error is below:
tbanboundtweets <- bind_tweets('tban_tweets', verbose = TRUE, output_format = "tidy")
**Error**: There are no files matching the pattern `^users_` in the specified directory
Thank you.
**`r tweets <- function(...)`**
#academictwitteR# #197 #204 #91

**Comment by cjbarrie** [comment, marked as answer]
This can only mean that your working directory is not set to where the tban_tweets files are stored. 

Check they are stored where you are pointing the bind_tweets function by doing 

`list.files('tban_tweets')`

**Reply by psalmuel19** [reply]
Thank you so much Chris
